src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml

```diff
@@ -143,3 +143,11 @@ ra_location:
     requirements:
         raLocationId: ".+"
     condition: "request.headers.get('Accept') matches '/^application\\\\/json($|[;,])/'"
+
+institution_configuration_options_for_institution:
+    path:    /institution-configuration-options/{institution}
```

Comment:
I agree with this. Perhaps we offer an `GET \institution-configuration\{institution}`, which contains the options and the RA locations? 

It feels less weird to me: the options and the RA locations are related to the concept of configuration regarding an institution. However, they _are_ separate concepts. Also, the RA locations already have a separate API, so I guess you'd either have two ways to get RA locations or change the RA location API to fall under `\institution-configuration\`.